airflow/www/views.py

```diff
@@ -2280,10 +2280,13 @@ class VariableView(wwwutils.DataProfilingMixin, AirflowModelView):
     verbose_name_plural = "Variables"
     list_template = 'airflow/variable_list.html'
 
-    def hidden_field_formatter(view, context, model, name):
```

Comment:
I thought it would make more sense since it's not just hidden field formatter, but basically a field formatter to handle errors as well